czkawka_core/src/bad_extensions.rs

```diff
@@ -145,9 +145,9 @@ static WORKAROUNDS: &[(&str, &str)] = &[
     ("html", "svg"),
     ("xml", "html"),
     // Probably bug in external library
-    ("msi", "ppt"), // Not sure whe ppt is not recognized
-    ("msi", "doc"), // Not sure whe doc is not recognized
-    ("exe", "xls"), // Not sure whe xls is not recognized
+    ("msi", "ppt"), // Not sure when ppt is not recognized
+    ("msi", "doc"), // Not sure when doc is not recognized
+    ("exe", "xls"), // Not sure when xls is not recognized
```

Comment:
In regard to the comment in line 147, this should probably say "why" rather than "when".